python_modules/libraries/dagster-databricks/dagster_databricks/ops.py

```diff
@@ -135,3 +138,224 @@ def create_ui_url(databricks_client, op_config):
+    ins: Optional[Mapping[str, In]] = None,
+    out: Optional[Union[Out, Mapping[str, Out]]] = None,
+) -> OpDefinition:
+    """
+    Creates an op that launches an existing databricks job.
+
+    As config, the op accepts a blob of the form described in Databricks' Job API:
+    https://docs.databricks.com/api-explorer/workspace/jobs/runnow. The only required field is
+    ``job_id``, which is the ID of the job to be executed. Additional fields can be used to specify
+    override parameters for the Databricks Job.
+
+    Arguments:
+        databricks_job_id (int): The ID of the Databricks Job to be executed.
+        databricks_job_configuration (dict): Configuration for triggering a new job run of a
+            Databricks Job. See https://docs.databricks.com/api-explorer/workspace/jobs/runnow
+            for the full configuration.
+        ins (Optional[Dict[str, In]]): Keyword argument passed to the underlying op.
+        out (Optional[Union[Out, Dict[str, Out]]]): Keyword argument passed to the underlying op.
+
+    Returns:
+        OpDefinition: An op definition to run the Databricks Job.
+
+    Example:
+        .. code-block:: python
+
+            from dagster import job
+            from dagster_databricks import create_databricks_run_now_op, databricks_client
+
+            DATABRICKS_JOB_ID = 1234
+
+
+            run_now_op = create_databricks_run_now_op(
+                databricks_job_id=DATABRICKS_JOB_ID,
+                databricks_job_configuration={
+                    "python_params": [
+                        "--input",
+                        "schema.db.input_table",
+                        "--output",
+                        "schema.db.output_table",
+                    ],
+                },
+            )
+
+            @job(
+                resource_defs={
+                    "databricks": databricks_client.configured(
+                        {
+                            "host": {"env": "DATABRICKS_HOST"},
+                            "token": {"env": "DATABRICKS_TOKEN"}
+                        }
+                    )
+                }
+            )
+            def do_stuff():
+                run_now_op()
+    """
+
+    @op(
+        ins=(ins or {"start_after": In(Nothing)}),
+        out=out,
+        config_schema={
+            "poll_interval_sec": Field(
+                float,
+                description="Check whether the Databricks Job is done at this interval.",
+                default_value=_DEFAULT_POLL_INTERVAL,
+            ),
+            "max_wait_time_sec": Field(
+                float,
+                description=(
+                    "If the Databricks Job is not complete after this length of time, raise an"
+                    " error."
+                ),
+                default_value=_DEFAULT_RUN_MAX_WAIT_TIME_SEC,
+            ),
+        },
+        required_resource_keys={"databricks"},
+        tags={"kind": "databricks"},
```

Comment:
in terms of parameterizing config, what I think *would* make sense is to make these default values parameterizable, which is right way to get "best of both worlds" here.

Reply:
And I think the approach is _preferable_ to calling `configured`

Reply:
Addressed in https://github.com/dagster-io/dagster/pull/12492/commits/9c42bfdc217cc32e0755af7eca2accb921765470